src/code/z_scene_table.c

```diff
@@ -1,6 +1,8 @@
 #include "global.h"
 #include "quake.h"
 
+#include "assets/scenes/indoors/souko/souko_scene.h"
```

Comment:
not quite related to your changes but the `#include "assets/scenes/indoors/miharigoya/miharigoya_scene.h"` seems oddly positioned in this file

Reply:
hm what would be a better positioning?

Reply:
the list seems to attempt to sort by category, but miharigoya is indoors, not dungeon. you can move it up to the indoors section you just created

Reply:
oh I see, done